scripts/run_command_building.py

```diff
@@ -14,33 +20,22 @@ def find_exe_path(path_to_search_exe_in, exe_identifier):
             return path_item.resolve()
     raise RuntimeError("Missing {searched_exe}")
 
-def make_command_to_run(path_where_to_find_exe, batch_name, study_path):
+def make_command_to_run(exe_path, batch_name, study_path):
```

Comment:
**make_command_to_run** function is now simplified.
Code about finding the executable path was moved elsewhere (into the main program)